### PR TITLE
Add a Less method unit test

### PR DESCRIPTION
**Pull request by dlion**
I saw there is a lack of unit test so I started to contribute to it creating just one, specifically for the `Less` method in which we can find some logic that I think worth to be tested.

ps: can you add the #hacktoberfest label to the project or sign this PR with the label `hacktoberfest-accepted` in order to be considered eligible for it? Thanks

### CODE REVIEW

**Comment by muesli** [comment]
Thanks for the PR @dlion! I'll review it this weekend. In the meantime I've already added the `hacktoberfest` label to our repos.

**Comment by dlion** [comment]
Hi @muesli , any news about this PR?

File: ui/markdown_test.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+	b := charm.Markdown{
+		ID:           1,
+		EncryptKeyID: "blabla2",
+		Note:         "Blabla2",
+		Body:         "blabla2",
+		CreatedAt:    time.Now().Local(),
+	}
+
+	t.Run("Should returns true if the first markdown is local and the second is not local", func(f *testing.T) {
```
Comment (muesli) [inline ui/markdown_test.go:27]: Typo/grammar: `Should return ...` throughout the file.

File: ui/markdown_test.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+		markdowns = append(markdowns, &markdown{
+			stashedMarkdown,
+			"blabla2",
+			"blabla2",
+			b,
+		})
+
+		if ok := markdowns.Less(0, 1); !ok {
+
```
Comment (muesli) [inline ui/markdown_test.go:45]: Spurious newline.

File: ui/markdown_test.go [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+		EncryptKeyID: "blabla2",
+		Note:         "Blabla2",
+		Body:         "blabla2",
+		CreatedAt:    time.Now().Local(),
+	}
+
+	t.Run("Should returns true if the first markdown is local and the second is not local", func(f *testing.T) {
+		var markdowns markdownsByLocalFirst
+
+		markdowns = append(markdowns, &markdown{
```
Comment (muesli) [inline ui/markdown_test.go:30]: It may be worth using a table-driven approach here, which could slim down the code by quite a bit.

**Comment by caarlos0** [comment]
this was removed from master.

thanks for the PR, sorry for the delay

**Pull request closed without merging**